proto/obb/markets.proto

```diff
@@ -6,16 +6,21 @@ option go_package = "oddin.gg/obb";
 option java_package = "com.oddin.obb";
 
 // MarketRequest is used as request for Markets RPC call
-message MarketRequest {
+message AvailableEventMarketsRequest {
```

Comment:
I suggest to drop Event and make it shorter as `event_id` is already argument. 
```suggestion
message AvailableMarketsRequest {
```

(If applied. Also for other `Availabe*` messages.)

Reply:
Yeas we can do it but i will rather use event_urn instead of ID because `od:match:1234` is urn not ID

Reply:
Renamed to `AvailableMarketsRequest`, used `event_urn` instead of `event_id`